llpc/test/shaderdb/OpFunction_TestDontInline_lit.spvasm

```diff
@@ -0,0 +1,38 @@
+; BEGIN_SHADERTEST
```

Comment:
You can remove "_lit" postfix. In the future, we will remove all such postfixes.